### PR TITLE
Add a stub file for Sydney

### LINKED ISSUE #14: runoxxvi recommendations for things to do in Sydney
Things to do in Sydney:

- [ ]  Eat different types of food.
- [ ]  beaches
- [ ]  bridge

### PR DESCRIPTION
**Pull request by runoxxvi**
This closes #14 

Adding a file with recommendations for Sydney

Hope this is OK, @githubteacher .

### CODE REVIEW

File: runoxxvi-sydney.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Sydney
+
+## Restaurants
+
```
Comment (Rachel2046) [inline runoxxvi-sydney.txt:5]: What is that? Make a real commendation!

Comment (Rachel2046) [inline runoxxvi-sydney.txt:5]: What is that? Make a real commendation! :sparkle:

Comment (Rachel2046) [inline runoxxvi-sydney.txt:5]: What is that? Make a real commendation! :sparkle:

Comment (Rachel2046) [inline runoxxvi-sydney.txt:5]: :taco:

Comment (Rachel2046) [inline runoxxvi-sydney.txt:5]: :taco:

File: runoxxvi-sydney.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Sydney
+
+## Restaurants
```
Comment (Rachel2046) [inline runoxxvi-sydney.txt:3]: :taco:

**Review by Rachel2046 — APPROVED** [review]

File: runoxxvi-sydney.txt (hunk 1 above)
Comment (runoxxvi) [reply runoxxvi-sydney.txt:5]: Sorry, I'll try harder next time.

Comment (runoxxvi) [reply runoxxvi-sydney.txt:5]: Stop repeating yourself!

Comment (runoxxvi) [reply runoxxvi-sydney.txt:5]: This is all too much.

Comment (runoxxvi) [reply runoxxvi-sydney.txt:5]: :taco: :sparkles:

**Pull request merged**